### PR TITLE
GH-1847 Viz II Summary

### PR DESCRIPTION
**Pull request by GordonSmith**
Allow user to pick columns
Allow embeded HTML

Fixes GH-1847

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage @mlzummo Please review.

File: src/chart/Summary.js [hunk 1]
```diff
@@ -84,23 +131,25 @@
...
         this._headerDiv
-            .style("color", this.colorStroke())
-            .text(row[1])
+            .transition()
+            .style("color", row.stroke)
+            [this.valueHTML() ? "html" : "text"](row.value)
         ;
         this._textDiv
-            .text(row[0])
+            [this.valueHTML() ? "html" : "text"](row.label)
```
Comment (jchambers-ln) [inline src/chart/Summary.js]: do you want to use a local var here or a constant for "text" and "html"?

File: src/chart/Summary.js [hunk 2]
```diff
@@ -84,23 +131,25 @@
...
                 "min-width": this.minWidth_exists() ? this.minWidth() + "px" : null,
                 "min-height": this.minHeight_exists() ? this.minHeight() + "px" : null
             })
         ;
         this._headerDiv
-            .style("color", this.colorStroke())
-            .text(row[1])
+            .transition()
+            .style("color", row.stroke)
+            [this.valueHTML() ? "html" : "text"](row.value)
```
Comment (jchambers-ln) [inline src/chart/Summary.js]: do you want to use a local var here or a constant for "text" and "html"?

File: src/chart/Summary.js [hunk 3]
```diff
@@ -116,13 +165,14 @@
...
-            .attr("style", "background-color:" + d3.rgb(this.colorFill()).darker(0.75))
+            .transition()
+            .style("background-color", d3.rgb(row.fill).darker(0.75))
         ;
         moreDivs.select("i")
             .attr("class", "fa " + this.moreIcon())
         ;
         moreDivs.select("span")
-            .text(this.moreText())
+            [this.moreTextHTML() ? "html" : "text"](function (d) { return d; })
```
Comment (jchambers-ln) [inline src/chart/Summary.js]: do you want to use a local string here or a constant for "text" and "html"?

**Comment by jchambers-ln** [comment]
see inline otherwise good

**Comment by GordonSmith** [comment]
@jchambers-ln Suggested changes made...

**Pull request merged**